src/templates/en/2020/stories/web-almanac-2020-page-content/index.html

```diff
@@ -0,0 +1,34 @@
+			<script type="application/json">
+				{"vars":{"gtag_id":"UA-22381566-3","config":{"UA-22381566-3":{"groups":"default"}}},"triggers":{"storyProgress":{"on":"story-page-visible","request":"event","vars":{"event_name":"custom","event_action":"story_progress","event_category":"${title}","event_label":"${storyPageIndex}","event_value":"${storyProgress}","send_to":"UA-22381566-3"}},"storyEnd":{"on":"story-last-page-visible","request":"event","vars":{"event_name":"custom","event_action":"story_complete","event_category":"${title}","event_label":"${storyPageCount}","send_to":"UA-22381566-3"}},"trackFocusState":{"on":"story-focus","tagName":"a","request":"click ","vars":{"event_name":"custom","event_action":"story_focus","event_category":"${title}","send_to":"UA-22381566-3"}},"trackClickThrough":{"on":"story-click-through","tagName":"a","request":"click ","vars":{"event_name":"custom","event_action":"story_click_through","event_category":"${title}","send_to":"UA-22381566-3"}},"storyOpen":{"on":"story-open","request":"event","vars":{"event_name":"custom","event_action":"story_open","event_category":"${title}","send_to":"UA-22381566-3"}},"storyClose":{"on":"story-close","request":"event","vars":{"event_name":"custom","event_action":"story_close","event_category":"${title}","send_to":"UA-22381566-3"}},"audioMuted":{"on":"story-audio-muted","request":"event","vars":{"event_name":"custom","event_action":"story_audio_muted","event_category":"${title}","send_to":"UA-22381566-3"}},"audioUnmuted":{"on":"story-audio-unmuted","request":"event","vars":{"event_name":"custom","event_action":"story_audio_unmuted","event_category":"${title}","send_to":"UA-22381566-3"}},"pageAttachmentEnter":{"on":"story-page-attachment-enter","request":"event","vars":{"event_name":"custom","event_action":"story_page_attachment_enter","event_category":"${title}","send_to":"UA-22381566-3"}},"pageAttachmentExit":{"on":"story-page-attachment-exit","request":"event","vars":{"event_name":"custom","event_action":"story_page_attachment_exit","event_category":"${title}","send_to":"UA-22381566-3"}},"storyBookendEnter":{"on":"story-bookend-enter","request":"event","vars":{"event_name":"custom","event_action":"story_bookend_enter","event_category":"${title}","send_to":"UA-22381566-3"}},"storyBookendExit":{"on":"story-bookend-exit","request":"event","vars":{"event_name":"custom","event_action":"story_bookend_exit","event_category":"${title}","send_to":"UA-22381566-3"}}}}			</script>
```

Comment:
Whose GA is this?

Reply:
I thought I added the one from the Web Almanac site? Let me double check if not

Reply:
Oh ok I didn't know you had grabbed it from our site, all good then. I thought maybe it was a default from the editor. I hadn't checked to see if it was our ID first.